### PR TITLE
Refactor feature section with parallax effectFeature section refactor

### LINKED ISSUE #1674: [feature]: Refactor feature section with parallax effect
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

currently users are not able to identify that the feature section has 3 tabs and they can click on them. 

So the feature sections needs to be expanded within 3 sub-sections divided in 2 parts. With feature heading and description on one side and image on the other.  I personally like how appsmith has implemented it under "Dev-first workflows" section with parallax effect. 

### Why should this be worked on?

better UX

### Repository

website

### PR DESCRIPTION
**Pull request by sahadat-sk**
Fixes:  [#1674](https://github.com/keploy/keploy/issues/1674)

Demo (Sorry for laggy video 😅):
[keploy.webm](https://github.com/keploy/website/assets/98450585/341d3935-791c-42e1-a159-f61266fa78ed)

### CODE REVIEW

**Review by nehagup — CHANGES_REQUESTED** [review]
good initiative @sahadat-sk ! 

- There's a lot of empty space over the first feature. 
- Once you scroll to the second feature the GIF transition should happen once the text of second feature has come to the middle of the first image. 
- In the small screen size this is completely breaking, please check the responsiveness. 
Screenshot 2024-03-13 at 7 57 51 PM

**Comment by sahadat-sk** [comment]
Hey @nehagup can you review now?

**Review by Hermione2408 — COMMENTED** [review]
@sahadat-sk Please revert back the linting changes

**Comment by Hermione2408** [comment]
Hey @sahadat-sk Smooth transition kudos! 

- Please keep the Heading Intact and inside the view when the transition is happening.
- Remove the empty space above first feature   
- Also I noticed you have removed the text "Give your teams the tool they need to move faster Accelerate development with streamlined testing, capturing network calls, automating scenarios, and seamless tool integration." try to accommodate it

**Comment by sahadat-sk** [comment]
Hey @Hermione2408, I have made the requested changes, can you review them now?

File: components/features.tsx [hunk 1]
```diff
@@ -43,340 +95,294 @@ export default function Features() {
...
+                alt="Features bg"
+              />
+              {/*<Image className="absolute left-0 w-full transform md:max-w-none animate-float" src={FeaturesElement} width={500} height="44" alt="Element" style={{ top: '30%' }} />*/}
+              {/*<div className="absolute left-0 w-full text-left transform md:max-w-none animate-float" style={{ top: '100%' }}><CopyButton codeToCopy={"curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh"} /></div>*/}
+            </div>
           </div>
         </div>
-      </div>
-    </section>
+        <div className="max-w-6xl mx-auto mb-24 -mt-24 text-center">
```
Comment (Hermione2408) [inline components/features.tsx]: Hey @sahadat-sk everything looks fine just remove this division it is not going well with the UI

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

File: components/features.tsx (hunk 1 above)
Comment (sahadat-sk) [reply components/features.tsx]: Done

**Review by Hermione2408 — APPROVED** [review]
LGTM @nehagup

**Review by nehagup — CHANGES_REQUESTED** [review]
Thanks for the PR @sahadat-sk ! A few changes.

- The section is breaking on md and sm screen sizes, please stick to the same old format for md and sm screen size.
- User needs to much scrolling from 1 feature to other, the second feature should start to appear as the first feature text comes to the top of container (attached screenshot)
Screenshot 2024-03-19 at 4 47 03 PM

- can you please center align the 3rd GIF in center?
- there's a lot of empty space after 3rd feature ends. Please reduce that to margins same as margins in other sections.
 
Screenshot 2024-03-19 at 4 54 57 PM

Rest looks good!

**Comment by sahadat-sk** [comment]
> Thanks for the PR @sahadat-sk ! A few changes.
> 
>     * The section is breaking on md and sm screen sizes, please stick to the same old format for md and sm screen size.
> 
>     * User needs to much scrolling from 1 feature to other, the second feature should start to appear as the first feature text comes to the top of container (attached screenshot)
> 
> 
> Screenshot 2024-03-19 at 4 47 03 PM
> 
>     * can you please center align the 3rd GIF in center?
> 
>     * there's a lot of empty space after 3rd feature ends. Please reduce that to margins same as margins in other sections.
> 
> 
> Screenshot 2024-03-19 at 4 54 57 PM
> 
> Rest looks good!

I have made the changes, can you please review them!!

**Review by nehagup — COMMENTED** [review]
@sahadat-sk Everything else looks good! Just that First feature fades away before the first GIF comes even to the center. It's fine to remove the 3rd GIF and scroll the features-text with just 2 GIFs so that text gets more time and GIFs doesn't change too soon.

**Comment by sahadat-sk** [comment]
> @sahadat-sk Everything else looks good! Just that First feature fades away before the first GIF comes even to the center. It's fine to remove the 3rd GIF and scroll the features-text with just 2 GIFs so that text gets more time and GIFs doesn't change too soon.

Hey @nehagup, I have made the changes, do let me know if there is anything else I need to change!

**Review by nehagup — APPROVED** [review]

**Pull request merged**